t/op/sub_lval.t

```diff
@@ -1096,3 +1096,25 @@ is "@119797", "4 5 6", 'writing to array returned by bare block';
     junkreturn = "b";
     is($x, "b", "junkreturn");
 }
+
+# v5.18 fails to return correct value from lvalue subroutine which returns
+# vec() and are called twice in rvalue context of one expression
```

Comment:
`are called twice` should be `is called twice`, as the subject of "are called" is `subroutine`, which is singular.

Reply:
Thank you very much for pointing this out.  I've pushed 853a6a9a936aa1c945522bacc1947a93e3cdd735 to fix this.